src/test/java/reposense/parser/ArgsParserTest.java

```diff
@@ -57,23 +58,25 @@ public void parse_withExtraWhitespaces_success() throws ParseException, IOExcept
     }
 
     @Test
-    public void parse_configFileOnly_success() throws ParseException, IOException {
+    public void parse_configFileOnly_success() throws ParseException, IOException, java.text.ParseException {
         Path expected = CONFIG_FILE_ABSOLUTE;
         String input = String.format("-config %s", CONFIG_FILE_ABSOLUTE);
         CliArguments cliArguments = ArgsParser.parse(translateCommandline(input));
         Files.isSameFile(expected, cliArguments.getConfigFilePath());
         //Optional arguments have default values
         Assert.assertEquals(Optional.empty(), cliArguments.getSinceDate());
         Assert.assertEquals(Optional.empty(), cliArguments.getUntilDate());
-        Assert.assertTrue(Files.isSameFile(Paths.get("."), cliArguments.getOutputFilePath()));
+        //Checks if generate output folder is in yyyy-MM-dd-HH-mm-ss format
+        Constants.DEFAULT_REPORT_NAME_FORMAT.parse(cliArguments.getOutputFilePath().getFileName().toString());
 
         input = String.format("-config %s", CONFIG_FILE_RELATIVE);
         cliArguments = ArgsParser.parse(translateCommandline(input));
         Files.isSameFile(expected, cliArguments.getConfigFilePath());
         //Optional arguments have default values
         Assert.assertEquals(Optional.empty(), cliArguments.getSinceDate());
         Assert.assertEquals(Optional.empty(), cliArguments.getUntilDate());
-        Assert.assertTrue(Files.isSameFile(Paths.get("."), cliArguments.getOutputFilePath()));
+        //Checks if generate output folder is in yyyy-MM-dd-HH-mm-ss format
```

Comment:
`Checks if output folder generated is in correct format`?
whitespace after `//` too